readme.rst

```diff
@@ -94,16 +94,16 @@ or from public sources on the internet.
 Supported Platforms
 ^^^^^^^^^^^^^^^^^^^
 PyNE is currently built and tested daily on the following platforms
-on Python 2.7
+on Python 3.5+
 
- * Ubuntu 12.04 - x86_64
+ * Ubuntu 18.04 - x86_64
  * OSX 10.8 - x86_64
 
-PyNE has pre-built binaries for the lastest release (0.4) on windows
-mac and linux 
+PyNE has pre-built binaries for the lastest release (0.7) on windows
```

Comment:
```suggestion
PyNE has pre-built binaries for the lastest release (0.7) on 
```
removing windows reference

Reply:
not sure if this suggestion overlaps with @nsryan2 #1285 where he changes the readme.